app/main/posts/detail/post-detail-data.controller.js

```diff
@@ -0,0 +1,222 @@
+module.exports = [
+    '$scope',
+    '$rootScope',
+    '$translate',
+    '$q',
+    '$filter',
+    '$location',
+    'PostEndpoint',
+    'CollectionEndpoint',
+    'UserEndpoint',
+    'TagEndpoint',
+    'FormAttributeEndpoint',
+    'FormStageEndpoint',
+    'FormEndpoint',
+    'Maps',
+    '_',
+    'Notify',
+    'moment',
+    'PostSurveyService',
+    '$state',
+    '$transition$',
+function (
+    $scope,
+    $rootScope,
+    $translate,
+    $q,
+    $filter,
+    $location,
+    PostEndpoint,
+    CollectionEndpoint,
+    UserEndpoint,
+    TagEndpoint,
+    FormAttributeEndpoint,
+    FormStageEndpoint,
+    FormEndpoint,
+    Maps,
+    _,
+    Notify,
+    moment,
+    PostSurveyService,
+    $state,
+    $transition$) {
+    $scope.$watch('post', function (post) {
+        activate();
+    });
+
+    $rootScope.setLayout('layout-d');
+    $scope.post = $scope.$resolve.post;
```

Comment:
this gets the post from the uirouter resolve and places it in the scope. Noting it be' onecause its of the things that changes in many places and maybe not everyone is super aware